src/sql/azdata.d.ts

```diff
@@ -189,6 +189,22 @@ declare module 'azdata' {
 		 * @param connectionProfile connection profile
 		 */
 		export function connect(connectionProfile: IConnectionProfile, saveConnection?: boolean, showDashboard?: boolean): Thenable<ConnectionResult>;
+
+		export type ConnectionEventType =
```

Comment:
Please add full doc comments for everything being added (including parameters, return values if any)